### PR TITLE
Add an endpoint, /_version, that describes the deployed application version.

### PR DESCRIPTION
**Pull request by jfhamlin**
Also removes some lingering react-starter-kit files.

### CODE REVIEW

File: tools/docker-build.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+COMMIT=${TRAVIS_COMMIT::8}
+SANITIZED_BRANCH=$(echo $TRAVIS_BRANCH|sed 's|/|-|g')
+REPO=sheltertechsf/askdarcel-web
+
+docker login -e $DOCKER_EMAIL -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+
+if [[ -n "$TRAVIS_TAG" ]]; then
+    TAG="$TRAVIS_TAG"
+else
+    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```
Comment (jfhamlin) [inline tools/docker-build.sh]: Looks like I've got tabs in here, but just want spaces. Will fix.

**Comment by lelandg559** [comment]
http://askdarcel.lvh.me:8080/_version redirects back to the home page for me, is that the correct path?

**Comment by jfhamlin** [comment]
It only affects CI and the docker image, so it won't affect the local server. You should know what version you're running in local dev :-P.

**Comment by lelandg559** [comment]
LGTM

**Pull request merged**